src/www/app/electron_main.ts

```diff
@@ -92,12 +92,13 @@ main({
   hasDeviceSupport: () => {
     return isOsSupported;
   },
-  getPersistentServerFactory: () => {
-    return (serverId: string, config: ServerConfig, eventQueue: EventQueue) => {
+  getServerFactory: () => {
+    return (serverId: string, serverName: string, config: ShadowsocksConfig,
```

Comment:
Revert serverName. See my other comment for explanation.

Reply:
Done in 76bde1c.